src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,52 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            return null;
```

Comment:
do not return null from methods

Reply:
You should fix (or discuss in chat if disagree) all mentor's comments before re-request new review. Be attentive and do not hesitate to contact us in MA chat if you do not understand something.